packages/core/src/ready-manager.ts

```diff
@@ -31,5 +31,6 @@ declare global {
 	interface Window {
 		ready: boolean;
 		getStaticCompositions: () => TCompMetadata[];
+		remotion_setFrame: (frame: number) => void
```

Comment:
Maybe rename all global variables to `__REMOTION_{variable}` , because it's more safe